### DISCUSSION TITLE
freezing pans (justlike in Excel) for the tab component

### DISCUSSION THREAD

**Discussion by alexisrc1** [sqlpage/SQLPage#682] [General]
It would be amazing for data analysis if there was a way to freeze columns and/or rows,so when tables become big it is easy to navigate them.
If you can add 2 two top-level parameters so we can indicate the numbers of rows / columns to freeze, for instance:

select 
    'table' as component,
2 as freeze_columns -- for freezing the first 2 columns of the table,
1 as freeze_rows -- for freezing the top row of the table (the row with columns names)
;
select 
    1 as a,
    2 as b;
select 
    3 as a,
    4 as b; 

Thanks 😉

**Comment by lovasoa** [comment]
I opened an issue to track the feature request and possible solutions here: https://github.com/sqlpage/SQLPage/issues/683
